### PR TITLE
Support podcast episodes in web-core

### PR DESCRIPTION
**Pull request by jankrcal**
This PR adds support for podcast episodes in web-core. To this end, it:
 1) extends Makefile etc. to process podcast images;
 2) adds a symlink to have includes from 2050;
 3) expands search.json to support episodes;
 4) tweaks css and includes here and there to render episode blocks for the web and for the search results.

This includes a few hacks:
 - point 2) is needed because the pages are rendered even if "output: false", see https://github.com/jekyll/jekyll/issues/7504
     -> I am open to better solutions
 - redirecting to podcast site in 3) and 4) is not ideal because:
    - it uses a custom plugin
    - we need to replace all relevant instances of `page.url` by `page | get_url` (done for now)

For the record, all other repos using web-core need two more lines in `_config.local.yml` in `collections:`:
```
  episodes:  # required by web-core
    output: false
```

### CODE REVIEW

**Review by mukrop — APPROVED** [review]
I've browsed through the changes on the web but unfortunately don't have time to test everything locally (it would be great if @mgrabovsky could, but you know if there's time for that).

In general, I think it's good (although now I see there were more complications than I expected). The few things I'd consider:
* Extend the comment in the podcast placeholder layout saying where the layout is used. (Is it currently only the block rendering code?)
* The adjustment with `get_url` is a bit clumsy, we may forget to do that in the future. Though I don't see a different option now :-/. If it stays this way, white this somewhere (in the Readme?).
* I was hesitant regarding the includes folder name (2050), why not "podcast"?
* Regarding the `output: false`, would not `published: false` work better? (I cannot find the full documentation now, maybe it does not exist as a global option.)

**Comment by jankrcal** [comment]
Thanks for the review and for the feedback!

> I've browsed through the changes on the web but unfortunately don't have time to test everything locally (it would be great if @mgrabovsky could, but you know if there's time for that).

There's no special pressure on my end w.r.t. this PR. If @mgrabovsky would find time within the next week for another round of feedback with local testing, I'd appreciate that. At the same time, I am confident enough to merge it without such a review.

> In general, I think it's good (although now I see there were more complications than I expected). The few things I'd consider:
> 
> * Extend the comment in the podcast placeholder layout saying where the layout is used. (Is it currently only the block rendering code?)

Done.

> * The adjustment with `get_url` is a bit clumsy, we may forget to do that in the future. Though I don't see a different option now :-/. If it stays this way, white this somewhere (in the Readme?).

Agreed. To simplify the review of this PR, I plan to replace all the current usages of `.url` by `| get_url` in a separate PR.

> * I was hesitant regarding the includes folder name (2050), why not "podcast"?

My rationale (for the sake of the podcast episode .md) was maximal brevity as the includes occur a lot. If you feel strongly about `podcast`, I'm happy to replace it.

> * Regarding the `output: false`, would not `published: false` work better? (I cannot find the full documentation now, maybe it does not exist as a global option.)

That works exactly the same in my local testing. Thus, keeping `output` as this is the documented option.

File: _includes/preview-block-small.html [hunk 1]
```diff
@@ -1,19 +1,19 @@
 {%- assign item = include.item %}
 {%- case item.layout %}
     {%- when 'infographic' %}
-        <img src="/assets/generated/{{ item.slug }}_600.png" alt="{{ item.title }}" class="img-fluid" />
+        <div style="background-image: url('/assets/generated/{{ item.slug }}_600.png')" class="specific-card"></div>
     {%- when 'dataset' %}
-        <img src="/assets/datasets/{{ item.slug }}.png" alt="{{ site.data.lang.text.dataset.dataset }}: {{ item.title }}" class="img-fluid card-img" />
+        <img style="background-image: url('/assets/datasets/{{ item.slug }}.png')" class="specific-card"></div>
```
Comment (mgrabovsky) [inline _includes/preview-block-small.html:6]:
```suggestion
        <div style="background-image: url('/assets/datasets/{{ item.slug }}.png')" class="specific-card"></div>
```

**Comment by jankrcal** [comment]
@mukrop Merging now but feel free to push for `_includes/podcast` instead of `_includes/2050` if you feel strongly. I can change it in a follow-up.

**Pull request merged**